ui/src/app/model/portCall/portcall-timestamp-type.enum.ts

```diff
@@ -18,3 +20,43 @@ export enum PortcallTimestampType {
   PTD_Berth = 'PTD-Berth',
   ATD_Berth = 'ATD-Berth'
 }
+
+export class TimestampTypeUtil {
+
+  static getPortcallTimestampTypes(publisherRole: PublisherRole): string[] {
+    switch (publisherRole) {
+      case PublisherRole.CA:
```

Comment:
I assume `AG` and `VSL` should go here too.  If in doubt, feel free to ask Guido.

Reply:
I talked with Charles Moret regarding roles and according to the table he sent me it doesn't seem like the remaining roles can create timestamps in the JIT (or is something we don't have to implement at the moment).

Reply:
Charles is as good an authority as Guido! Thanks for checking.